sample/HttpWorker/host.json

```diff
@@ -0,0 +1,9 @@
+{
+	"version": "2.0",
+	"httpInvoker": {
+		"description": {
```

Comment:
Why do we need the nested "description" block? I don't see our other config sections doing this - settings for the target are just top level.

Reply:
done!